### ISSUE TITLE
Emu crashes with game (Unofficial rom)

### ISSUE THREAD

**Issue by Aurumaker72** [TASVideos/BizHawk#2632]
[//]: # "This issue description supports Markdown syntax (this is what comments look like). There's a cheatsheet here: https://guides.github.com/features/mastering-markdown/"
[//]: # "You can leave these comments here or delete them. Also, please remember to search for similar issues before writing anything, including in closed issues!"
[//]: # "One more thing: if you're on Linux, please comment on #1430 instead of opening an issue so we don't annoy the other devs."

### Summary
Managed to crash the SMW nes pirate rom and it took BizHawk with it
This seems worrying because it completely crashes BizHawk so what happens if you're recording a movie?!!

### Repro
[//]: # "If you can't figure out the list of steps, delete this section and put 'heisenbug' in the summary somewhere. If a Lua script can cause the bug, you can embed that instead (as simple as possible please)."
1. Open the SMW NES remake rom (QuickNES, NESHawk, Probably happens on SubNESHawk too but I can't test it due to slowdown)
2. Crash the game with too many sprites
3. Look at what you did... 😐 

### Output
[//]: # "Paste the contents of the error dialog if there is one (try Ctrl+C, it usually works), or paste the output from the Lua Console, or delete this section."
```
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at BizHawk.Emulation.Cores.Nintendo.NES.Mapper090.ReadExp(Int32 addr)
```

### Host env.
[//]: # "List the computers you've found the bug with. If there's a version that doesn't have the bug, please put that in too. Here are some examples:"
- BizHawk 2.6.0
- Ryzen 3 3200G
- 6GB usable RAM

[//]: # "(screenshots, if applicable)"

[//]: # "That's it! If you'd like to help more, you could try a dev build (see Testing in the readme) or an older release. Click submit now and you can edit it later."

**Comment by YoshiRulz** [comment]
Would #2364 fix this? (That is, can Mesen run the romhack with that sprite limit setting?)

**Comment by Aurumaker72** [comment]
> Would #2364 fix this? (That is, can Mesen run the romhack with that sprite limit setting?) edit: also link us the patch and the hash of the base rom

Probably

**Comment by nattthebear** [comment]
This fix actually made it into 2.6.1

**Comment by Aurumaker72** [comment]
Oh cool
